src/django/api/models.py

```diff
@@ -259,11 +259,11 @@ class Source(models.Model):
 
     @property
     def display_name(self):
+        name = self.contributor.name \
+            if self.contributor else '[No Contributor]'
```

Comment:
Good question. We should actually use the same alternate string. I think "Unknown" is better, because there always has to be a contributor, at least historically. I will change this one to match.